### DISCUSSION TITLE
background images are not showing up

### DISCUSSION THREAD

**Discussion by wifv** [community/community#57167] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

page: https://wifv.github.io/yasmintour/
repo: https://github.com/wifv/yasmintour/
i don't know why but the first image is not showing up.
HELP.

image: yasmintour/assets/images/mekkah.jpg
css: yasmintour/assets/css/style.css   315

**Comment by RaoKamranshauket** [comment]
Image mekkah.jpg is not found in your code
secondly change the path first give me image you want to apper

**Comment by emilburganov** [comment]
You don't have this picture in your repository(

**Comment by mohammed-r-mohsen** [comment]
1- you don't have the img that you mention it 
2- the url in your style file should be like this background: url(../mekkah.jpg)no-repeat; not background: url(./mekkah.jpg)no-repeat;
